### PR TITLE
wireshark: version bump to 2.0.0

### PR DESCRIPTION
**Pull request by EppO**

### CODE REVIEW

File: Library/Formula/wireshark.rb [hunk 1]
```diff
@@ -3,9 +3,9 @@ class Wireshark < Formula
...
   stable do
-    url "https://www.wireshark.org/download/src/all-versions/wireshark-1.12.8.tar.bz2"
-    mirror "https://1.eu.dl.wireshark.org/src/wireshark-1.12.8.tar.bz2"
-    sha256 "357e0a4e49525d80cdc740bb16539fcdb526ad38cc2ed6cabedafc9bdee5c7e7"
+    url "https://www.wireshark.org/download/src/all-versions/wireshark-2.0.0.tar.bz2"
+    mirror "https://1.eu.dl.wireshark.org/src/wireshark-2.0.0.tar.bz2"
+    sha256 "90026c761a85701d7783c7e2eaa4c1de247dfbadbd53221df355f121e42691dd"
 
     # Removes SDK checks that prevent the build from working on CLT-only systems
     # Reported upstream: https://bugs.wireshark.org/bugzilla/show_bug.cgi?id=9290
```
Comment (dunn) [inline Library/Formula/wireshark.rb]: This patch is doesn't apply cleanly anymore:
```
==> Verifying wireshark-2.0.0.tar.bz2 checksum
tar xf /Library/Caches/Homebrew/wireshark-2.0.0.tar.bz2
==> Patching
patching file configure
Hunk #1 FAILED at 16703.
1 out of 1 hunk FAILED -- saving rejects to file configure.rej
Error: Failure while executing: /usr/bin/patch -g 0 -f -p1
```

**Comment by mkozjak** [comment]
Anyone still getting this?

```brew cask install --force --appdir=/Applications wireshark                
==> Satisfying dependencies
complete
==> Downloading https://www.wireshark.org/download/osx/Wireshark%201.12.7%20Intel%2064.dmg

curl: (22) The requested URL returned error: 404 Not Found
Error: Download failed on Cask 'wireshark' with message: Download failed: https://www.wireshark.org/download/osx/Wireshark%201.12.7%20Intel%2064.dmg```

**Comment by DomT4** [comment]
The Cask is a separate project, albeit one tied into the Homebrew ecosystem. You'd have to ask them for advice on Cask issues I'm afraid.

**Comment by rakkesh** [comment]
`wireshark` formula without GUI, i.e., `tshark` and other related stuff, seems to install fine without the patch. 

When I try to install with `qt5`, it fails even when qt5 is already installed. I had previously installed `qt5` when `Xcode` was installed on my computer. I guess, `Xcode` is a dependency for build `wireshark` GUI.

```
/u/l/L/Formula ❯❯❯ brew install --verbose --debug wireshark --with-qt5
/usr/local/Library/brew.rb (Formulary::FormulaLoader): loading /usr/local/Library/Formula/wireshark.rb
/usr/local/Library/brew.rb (Formulary::FormulaLoader): loading /usr/local/Library/Formula/pkg-config.rb
/usr/local/Library/brew.rb (Formulary::FormulaLoader): loading /usr/local/Library/Formula/glib.rb
/usr/local/Library/brew.rb (Formulary::FormulaLoader): loading /usr/local/Library/Formula/xz.rb
/usr/local/Library/brew.rb (Formulary::FormulaLoader): loading /usr/local/Library/Formula/gettext.rb
/usr/local/Library/brew.rb (Formulary::FormulaLoader): loading /usr/local/Library/Formula/libffi.rb
/usr/local/Library/brew.rb (Formulary::FormulaLoader): loading /usr/local/Library/Formula/gnutls.rb
/usr/local/Library/brew.rb (Formulary::FormulaLoader): loading /usr/local/Library/Formula/libtasn1.rb
/usr/local/Library/brew.rb (Formulary::FormulaLoader): loading /usr/local/Library/Formula/gmp.rb
/usr/local/Library/brew.rb (Formulary::FormulaLoader): loading /usr/local/Library/Formula/nettle.rb
/usr/local/Library/brew.rb (Formulary::FormulaLoader): loading /usr/local/Library/Formula/libgcrypt.rb
/usr/local/Library/brew.rb (Formulary::FormulaLoader): loading /usr/local/Library/Formula/libgpg-error.rb
/usr/local/Library/brew.rb (Formulary::FormulaLoader): loading /usr/local/Library/Formula/d-bus.rb
/usr/local/Library/brew.rb (Formulary::FormulaLoader): loading /usr/local/Library/Formula/geoip.rb
/usr/local/Library/brew.rb (Formulary::FormulaLoader): loading /usr/local/Library/Formula/c-ares.rb
/usr/local/Library/brew.rb (Formulary::FormulaLoader): loading /usr/local/Library/Formula/qt5.rb
qt5: A full installation of Xcode.app is required to compile this software.
Installing just the Command Line Tools is not sufficient.
Xcode can be installed from the App Store.
Error: An unsatisfied requirement failed this build.
/usr/local/Library/Homebrew/formula_installer.rb:282:in `check_requirements'
/usr/local/Library/Homebrew/formula_installer.rb:249:in `compute_dependencies'
/usr/local/Library/Homebrew/formula_installer.rb:172:in `install'
/usr/local/Library/Homebrew/cmd/install.rb:218:in `install_formula'
/usr/local/Library/Homebrew/cmd/install.rb:97:in `block in install'
/usr/local/Library/Homebrew/cmd/install.rb:97:in `each'
/usr/local/Library/Homebrew/cmd/install.rb:97:in `install'
/usr/local/Library/brew.rb:146:in `<main>'
/u/l/L/Formula ❯❯❯
/u/l/L/Formula ❯❯❯
/u/l/L/Formula ❯❯❯ brew info qt5
qt5: stable 5.5.1 (bottled), HEAD [pinned at 5.5.1, keg-only]
Version 5 of the Qt framework
https://www.qt.io/
/usr/local/Cellar/qt5/5.5.1 (6930 files, 165M)
  Built from source with: --without-webengine --without-webkit
From: https://github.com/Homebrew/homebrew/blob/master/Library/Formula/qt5.rb
==> Dependencies
Build: xz ✔
Optional: d-bus ✔
```

**Comment by DomT4** [comment]
Closing in favour of https://github.com/Homebrew/homebrew/pull/46819.

**Pull request closed without merging**